packages/web-runtime/src/components/Topbar/Notifications.vue

```diff
@@ -22,7 +22,10 @@
       class="oc-overflow-auto oc-width-3-4 oc-width-large@s"
       padding-size="small"
     >
-      <div v-for="(el, index) in activeNotifications" :key="index" class="oc-width-1-1">
+      <div v-if="!activeNotifications.length">
+        <span v-text="$gettext('Nothing new')" />
+      </div>
+      <div v-for="(el, index) in activeNotifications" v-else :key="index" class="oc-width-1-1">
```

Comment:
Please wrap the `div` into a `template` tag and put the `v-else` to the template. The directives `v-for` and `v-if`/`v-else` are never to be used together on the same element. See https://eslint.vuejs.org/rules/no-use-v-if-with-v-for.html for details.

Reply:
I know but linter didn't complain this time thats why i though it was fine, i think v-else is actually okay for v-for. Still don't get why an extra wrapper is needed but okay to me

Reply:
changed it.